09-Time-Series-Data.ipynb

```diff
@@ -0,0 +1,786 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "lyra_lightcurve_fullday = lc.LYRALightCurve.create('2011-06-07')\n",
+    "lyra_lightcurve = lyra_lightcurve_fullday.truncate('2011-06-07 06:00','2011-06-07 08:00')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Part of the advantage of using these inbuilt functions we can get a quicklook at our data using simple commands:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "fig = goes_lightcurve.peek()\n",
+    "fig = lyra_lightcurve.peek()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "### Accessing and using the data\n",
+    "\n",
+    "More custom plots can be made easily by accessing the data in the lightcurve object. Both the time information and the data are contained within the lightcurve.data object, which is a pandas dataframe. We can see what data is contained in the dataframe by finding which keys it contains and also asking what's in the meta data: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "print(lyra_lightcurve.data.keys())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "print(lyra_lightcurve.meta)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that the meta data information is stored in something called `OrderedDict`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<section class=\"objectives panel panel-info\">\n",
+    "<div class=\"panel-heading\">\n",
+    "<h2><span class=\"fa fa-thumb-tack\"></span> On Dictionaries </h2>\n",
+    "</div>\n",
+    "<br/>\n",
+    "We can create keyword-data pairs to form a dictionary (shock horror) of values. In this case we have defined some strings and number to represent temperatures accross europe\n",
+    "\n",
+    "<pre>\n",
+    "temps = {'Brussles': 9, 'London': 3, 'Barcelona': 13, 'Rome': 16}\n",
+    "temps['Rome']\n",
+    "16\n",
+    "</pre>\n",
+    "\n",
+    "We can also find out what keywords are associated with a given dictionary, In this case:\n",
+    "\n",
+    "<pre>\n",
+    "temps.keys()\n",
+    "dict_keys(['London', 'Barcelona', 'Rome', 'Brussles'])\n",
+    "</pre>\n",
+    "\n",
+    "</code>\n",
+    "<br/>\n",
+    "</section>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can use these keys to plot specific parameters from the lightcurve, Aluminium is 'CHANNEL_3' and Zirconium is 'CHANNEL_4"
```

Comment:
May (or may not, depends what the audience knows) be worth explaining quickly that the different data series in the object are measurements by different filters on the same instrument.